src/sentry/integrations/jira_server/client.py

```diff
@@ -102,58 +306,3 @@ def request(self, *args, **kwargs):
                 signature_type="auth_header",
             )
         return self._request(*args, **kwargs)
-
-
-class JiraServer:
```

Comment:
Same as my previous comment w/ `JiraCloud` - this was refactored with the removal of `jira_style`.